pwndbg/disasm/__init__.py

```diff
@@ -119,8 +119,31 @@ def get_disassembler(pc):
                                    pwndbg.arch.endian,
                                    extra)
 
+class SimpleInstruction(object):
+    def __init__(self, address):
+        self.address = address
+        instr = gdb.execute('x/2i %#x' % self.address, to_string=True)
```

Comment:
Can we use GDB Python API to disassemble instructions instead of gdb.execute that invokes eXamine command with instructions format?

See: https://sourceware.org/gdb/onlinedocs/gdb/Architectures-In-Python.html#Architectures-In-Python:~:text=Function%3A%20Architecture.disassemble

Reply:
It may help us to avoid all the weird strip/split/string formatting stuff

Tho I guess x/i may give us more information than pure disassembly? I am not sure, happy to hear about it, but maybe we can do it in a nicer way with proper APIs

Reply:
I didn't know the disassemble API before, I'll try to use it. Thanks.